flow/environment.py

```diff
@@ -88,7 +88,7 @@ def __init__(cls, name, bases, dct):
             cls.registry = OrderedDict()
         else:
             cls.registry[name] = cls
-        return super().__init__(name, bases, dct)
+        super().__init__(name, bases, dct)
```

Comment:
what is the difference between calling the init and returning it here?

Reply:
The return is just a mistake. The return value of an `__init__` method isn’t used — returning the result of the parent constructor did nothing here, but it was triggering a warning from pylint. (I was using pylint to help me analyze the flow codebase and catch potential issues that I fixed in this series of PRs.)

Reply:
Ah, OK! There are so many good tools... 🤩 I am just learning about many of them, and they are so useful!